FinSearchUnified/Subscriber/Frontend.php

```diff
@@ -219,4 +217,42 @@ private function isManufacturerPage(Request $request)
     {
         return $request->getControllerName() === 'listing' && $request->getActionName() === 'manufacturer';
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return bool
+     */
+    private function isLegacySearch(Request $request)
+    {
+        $params = $request->getQuery();
+
+        if ($params['controller'] === 'FinSearchAPI' && $params['action'] === 'search') {
```

Comment:
`return $params['controller'] === 'FinSearchAPI' && $params['action'] === 'search'` will do the same.